troop/views.py

```diff
@@ -233,3 +237,60 @@ def form_invalid(self, form):
         response = super().form_invalid(form)
         response.status_code = 422  # Unprocessable Entity
         return response
+
+
+def csv_participant_export(request, troop_number):
+    troop = user_managed_troop(request.user, troop_number)
+    if not troop:
+        raise PermissionDenied()
+
+    response = HttpResponse(content_type="text/csv")
+    response["Content-Disposition"] = 'attachment; filename="packmas13_{}.csv"'.format(
+        troop.number
+    )
+    writer = csv.writer(response)
+
+    fields = [
+        "first_name",
+        "last_name",
+        "gender",
+        "birthday",
+        "email",
+        "nami",
+        "age_section",
+        "is_leader",
+        "diet",
+        "medication",
+        "comment",
+        "attendance",
+    ]
+    attendance_days = [str(day.date) for day in Attendance.objects.all()]
+    fields.extend(attendance_days)
+
+    writer.writerow(fields)
+
+    for p in (
+        Participant.objects.filter(troop=troop.id)
+        .prefetch_related("diet", "attendance")
+        .all()
+    ):
+        writer.writerow(participant_row(fields, attendance_days, p))
+    return response
+
+
+def participant_row(fields, attendance_days, participant):
+    days = [str(day.date) for day in participant.attendance.all()]
+
+    for field in fields:
+        if field == "is_leader":
+            yield 1 if participant.is_leader else ""
+        elif field == "age_section":
+            yield section_trans(participant.age_section)
```

Comment:
```suggestion
            yield _(participant.age_section)
```

Reply:
`get_age_section_display` should be used here

Reply:
Yes, I thought so, too. But I got some weird output instead. Does it work for you?